### ISSUE TITLE
What is the exact syntax of M: header

### ISSUE THREAD

**Issue by bwl21** [moinejf/abc2svg#34]
One of my user had the request that meter changes within the piece. I also display the content of the M: header literally in the table harp sheets. So they had the need to express the multiple meters in the M: header.

Eventually this worked:

M: 3/4 (4/4)

But I do not understand why. When I use

M: 3/4 4/4

then the bar numbers in Zupfnoter are not correct and I had Problems to get the countnotes right on the very first measure. It seems that the property `wmeasure` gets a different value depending of the second value in paranthesis.

It also appears abc2svg produces bar numbers only if we have at least filled the measure.

I found the documentation in http://moinejf.free.fr/abcm2ps-doc/features.xhtml. But i do not know what exactly I can write in the M: line and what the effect will be. Even http://abcnotation.com/wiki/abc:standard:v2.2#mmeter does not state all the options of M:

Could you help me with some hints how to deal with these things. First I want to do it right on the ABC-level. When I understand this and the result in JSON, then I can adapt the harpnote transformation accordiingly.

**Comment by moinejf** [comment]
The M: syntax is very lazy: it is just a mix of digits, slashes, '+' and parenthesis (plus 'C', 'C|' - I added 'c' and 'o', w/ or w/o a dot).

abc2svg tries to put these elements the best it can on the staves. The numbers/signs on the left of the slashes go above the ones on the right. When no slash, they are centered.

Except for simple cases, the measure duration which is needed to put a number on the measure bars cannot be computed. So, I extended the M: syntax by an equal sign followed by the measure duration:

M:3/4 (4/4)=3/4
M:3/4 4/4=7/4
